monitoring/logicmonitor_facts.py

```diff
@@ -0,0 +1,633 @@
+#!/usr/bin/python
+
+"""LogicMonitor Ansible module for managing Collectors, Hosts and Hostgroups
+   Copyright (C) 2015  LogicMonitor
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software Foundation,
+   Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA"""
+
+
+import socket
+import sys
+import types
+import urllib
+
+HAS_LIB_JSON = True
+try:
+    import json
+    # Detect the python-json library which is incompatible
+    # Look for simplejson if that's the case
+    try:
+        if (
+         not isinstance(json.loads, types.FunctionType) or
+         not isinstance(json.dumps, types.FunctionType)
+        ):
+            raise ImportError
+    except AttributeError:
+        raise ImportError
+except ImportError:
+    try:
+        import simplejson as json
+    except ImportError:
+        print(
+            '\n{"msg": "Error: ansible requires the stdlib json or ' +
+            'simplejson module, neither was found!", "failed": true}'
+        )
+        HAS_LIB_JSON = False
+    except SyntaxError:
+        print(
+            '\n{"msg": "SyntaxError: probably due to installed simplejson ' +
+            'being for a different python version", "failed": true}'
+        )
+        HAS_LIB_JSON = False
+
+
+DOCUMENTATION = '''
+---
+module: logicmonitor_facts
+short_description: Collect facts about LogicMonitor objects
+description:
+    - LogicMonitor is a hosted, full-stack, infrastructure monitoring platform.
+    - This module collects facts about hosts abd host groups within your LogicMonitor account.
+version_added: "2.2"
+author: Ethan Culler-Mayeno, Jeff Wozniak
```

Comment:
Can you please change this to Array in the form First Last (@GitHub)

This will make it easier for when we get the Bot to start parsing Modules automatically